packages/node/src/index.ts

```diff
@@ -454,6 +454,8 @@ export const build: BuildV3 = async ({
       entrypoint: handler,
       files: preparedFiles,
 
+      ...(staticConfig?.regions && { regions: staticConfig?.regions }),
```

Comment:
```suggestion
      regions: staticConfig?.regions
```

Also, I think you can simplify this